server/src/main/java/org/elasticsearch/cluster/coordination/StableMasterHealthIndicatorService.java

```diff
@@ -103,7 +106,6 @@ public class StableMasterHealthIndicatorService implements HealthIndicatorServic
     public static final Setting<TimeValue> NODE_HAS_MASTER_LOOKUP_TIMEFRAME_SETTING = Setting.timeSetting(
         "health.master_history.has_master_lookup_timeframe",
         new TimeValue(30, TimeUnit.SECONDS),
-        new TimeValue(1, TimeUnit.SECONDS),
```

Comment:
Oh, I don't think we should do this. If this was relaxed for testing reasons I think we should look into finding a different way to test things?